<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDetailsDialog</class>
 <widget class="QDialog" name="AccountDetailsDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Account usage details</string>
  </property>
  <property name="windowIcon">
   <iconset theme="mega">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
#AccountDetailsDialog
{
    border-radius: 0 0 4px 4px;
    background-color: #FFFFFF;
}

/* Config all labels */
QLabel
{
    font-family: &quot;Lato&quot;;
}

#sHeader
{
    background-color: #F4F6F8;
    height: 96px;
    width: 440px;
    height: 96px;
    width: 440px;
    max-height: 96px;
    min-height: 96px;
    max-width:440px;
    min-width:440px;
}

#lCloudDrive, #lVault, #lRubbish, #lStorage, #lTransfer, #lAvailable
{
  color: #333333;
  font-size: 13px;
  font-weight: bold;
}

#lStorage, #lTransfer
{
  qproperty-alignment: 'AlignLeft | AlignVCenter';
}

#wCloudDrive, #wRubbish, #wVault
{
    height: 32px;
    max-height: 32px;
    min-height: 32px;
}

#wCloudDrive, #wRubbish, #wVault, QProgressBar, #lUsedCloudDrive, #lUsedInbox,  #lUsedRubbish, #lUsedByVersions, #lAvailableStorage, #lUsedVault
{
    margin-right: 24px;
}

#lStorage, #lTransfer
{
    padding-bottom: 4px;
}

#lCloudDrive, #lVault, #lRubbish, #lAvailable
{
    margin-left: 10px;
    qproperty-alignment: 'AlignLeft | AlignBottom';
}

QProgressBar
{
    margin-left: 12px;
}

#lUsedCloudDrive, #lUsedVault,  #lUsedRubbish, #lUsedByVersions, #lAvailableStorage
{
  color: #333333;
  font-size: 13px;
  padding-right: 4px;
}

#lUsedCloudDrive, #lUsedInbox, #lUsedRubbish, #lUsedVault
{
    qproperty-alignment: 'AlignRight | AlignBottom';
}

#lUsedByVersions, #lAvailableStorage
{
    qproperty-alignment: 'AlignRight | AlignVCenter';
}

#wDetailedUsage
{
    min-height: 120px;
}

#AccountDetailsDialog[storageState=&quot;warning&quot;] #wDetailedUsage QProgressBar::chunk
{
  background-color: #FF993B;
}

#AccountDetailsDialog[storageState=&quot;full&quot;] #wDetailedUsage  QProgressBar::chunk
{
  background-color: #FF5646;
}

#wDetailedUsage QPushButton
{
    margin-left: 24px;
    height: 32px;
    width: 32px;
    min-width: 32px;
    min-height: 32px;
    max-width: 32px;
    max-height: 32px;
    border-radius: 8px;
    background-color: #EEEEEE;
    qproperty-iconSize: 24px;
}

#bCloudDrive, #bVault, #bRubbish
{
    qproperty-iconSize: 24px;
}

#bCloudDrive
{
    qproperty-icon : url(:/images/account_details/Cloud_Drive_24_ico.png);
}

#bVault
{
    qproperty-icon : url(:/images/Backups_24_ico.png);
}

#bRubbish
{
    qproperty-icon : url(:/images/account_details/Bin_24_ico.png);
}

#bAvailable
{
    qproperty-icon : url(:/images/account_details/available_storage.png);
}

#lVersionIcon
{
    qproperty-icon : url(:/images/account_details/versions.png);
	padding-left:4px;
}

#lLoading
{
    border-radius: 16px;
    background-color: white;
    height: 32px;
    max-height: 32px;
    min-height: 32px;
    max-width:440px;
    min-width:10px;
    font-size: 14px;
    color: #333333;
    padding-left:24px;
    padding-right:24px;
    qproperty-alignment: 'AlignVCenter | AlignHCenter';
}

#pLoading, #pStorageAndTransferUsage
{
    height: 96px;
    width: 440px;
    max-height: 96px;
    min-height: 96px;
    max-width:440px;
    min-width:440px;
}

QProgressBar
{
    height: 4px;
    min-height: 4px;
    max-height: 4px;
    border:0px;
    border-radius: 2px;
    background-color: #EEEEEE;
    margin-bottom: 4px;
}

QProgressBar::chunk
{
  background-color: #6FD7FF;
  border-radius: 2px;
  border:0px;
}

#wSizeVersions
{
    background-color: #F2F2F2;
    border-radius: 8px;
    margin-left: 24px;
    margin-right: 24px;
    margin-bottom: 26px;
    height: 32px;
    max-height: 32px;
    min-height: 32px;
    border-top: 2px inset #ECECEC;
    border-left: 1px inset #ECECEC;
    border-right: 1px inset #ECECEC;
    border-bottom: 1px inset #ECECEC;
}

#lVersions
{
    color: #333333;
    font-size: 13px;
    padding-left: 8px;
}

#wSizeVersions QLabel
{
    margin-bottom: 26px;
}

#AccountDetailsDialog[loading=true]  #sHeader
{
    /* Loading page */
    qproperty-currentIndex: 0;
}

#AccountDetailsDialog[loading=false]  #sHeader
{
    /* Usage page */
    qproperty-currentIndex: 1;
}

#AccountDetailsDialog[accountType=&quot;free&quot;] #pStorageAndTransferUsage QStackedWidget, #AccountDetailsDialog[accountType=&quot;pro&quot;] #pStorageAndTransferUsage QStackedWidget
{
    /* Business account pages */
    qproperty-currentIndex: 0;
}

#AccountDetailsDialog[accountType=&quot;business&quot;]  #pStorageAndTransferUsage QStackedWidget
{
    /* Non-business account pages */
    qproperty-currentIndex: 1;
}

#sStorage, #sTransfer
{
    height: 52px;
    width: 56px;
    min-height: 52px;
    min-width: 56px;
    max-height: 52px;
    max-width: 56px;
    margin-left: 24px;
}

#wStorageUsage, #wTransferUsage
{
    height: 96px;
    width: 219px;
    max-height: 96px;
    min-height: 96px;
    max-width:219px;
    min-width:219px;
}

#wStorageUsage
{
    border-right: 1px solid white;
}

#wTransferUsage
{
    border-left: 1px solid white;
}

#pStorageAndTransferUsage QPushButton
{
    qproperty-iconSize: 44px;
    border:0px;
}

#AccountDetailsDialog[accountType=&quot;business&quot;] #bBusinessStorage
{
    qproperty-icon :url(:/images/storage_for_business.png);
}

#AccountDetailsDialog[accountType=&quot;business&quot;]  #bBusinessTransfer
{
    qproperty-icon :url(:/images/transfer_for_business.png);
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="layoutWidget">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sHeader">
     <widget class="QWidget" name="pLoading">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lLoading">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Loading...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pStorageAndTransferUsage">
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wStorageUsage" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="sStorage">
            <widget class="CircularUsageProgressBar" name="wCircularStorage"/>
            <widget class="QWidget" name="wBusinessStorage">
             <layout class="QHBoxLayout" name="horizontalLayout_24">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="bBusinessStorage"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wStorageDetails" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lStorage">
               <property name="text">
                <string>Storage</string>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lUsedStorage">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wTransferUsage" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="sTransfer">
            <widget class="CircularUsageProgressBar" name="wCircularTransfer"/>
            <widget class="QWidget" name="wBusinessTransfer">
             <layout class="QHBoxLayout" name="horizontalLayout_26">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="bBusinessTransfer"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wTransferDetails" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lTransfer">
               <property name="text">
                <string>Transfer</string>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lUsedTransfer">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wDetailedUsage" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="wDetailedUsageLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <widget class="QWidget" name="wCloudDrive" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bCloudDrive"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lCloudDrive">
               <property name="text">
                <string>Cloud drive</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lUsedCloudDrive"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="pbCloudDrive">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wVault" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bVault"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_30">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lVault">
               <property name="text">
                <string>Backups</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lUsedVault"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="pbVault">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wRubbish" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bRubbish"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lRubbish">
               <property name="text">
                <string>Rubbish bin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lUsedRubbish"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="pbRubbish">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wAvailable" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bAvailable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>32</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <property name="spacing">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="lAvailable">
             <property name="text">
              <string>Available storage</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lAvailableStorage"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wSizeVersions" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lVersionIcon">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lVersions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Of that, size taken up by file versions:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lUsedByVersions"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircularUsageProgressBar</class>
   <extends>QWidget</extends>
   <header>CircularUsageProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
